<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ANURAG" failures="1" tests="6" name="class3" time="0.035" errors="0" timestamp="2025-05-18T13:54:38 IST">
  <testcase classname="introduction.Introduction_To_TestNG_5" name="printData" time="0.001"/>
  <testcase classname="introduction.Introduction_To_TestNG_5" name="printData" time="0.002"/>
  <testcase classname="introduction.Introduction_To_TestNG_5" name="printData" time="0.001"/>
  <testcase classname="introduction.Introduction_To_TestNG_5" name="printLocalVariable" time="0.002"/>
  <testcase classname="introduction.Introduction_To_TestNG_5" name="testFailure" time="0.012">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at introduction.Introduction_To_TestNG_5.testFailure(Introduction_To_TestNG_5.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testFailure -->
  <testcase classname="introduction.Introduction_To_TestNG_5" name="print" time="0.001"/>
</testsuite> <!-- class3 -->
